internal/kapp/kapp.go

```diff
@@ -187,3 +221,53 @@ func (c *Client) Inspect(appName string) (string, error) {
 
 	return outBuf.String(), nil
 }
+
+// Inspect inspects a kapp app
+func (c *Client) Inspect(appName string) (string, error) {
+	return c.inspectWithFlags(appName, []string{"--json"})
+}
+
+// InspectWithTree inspects a kapp app with tree output showing resource hierarchy
+func (c *Client) InspectWithTree(appName string) (string, error) {
+	return c.inspectWithFlags(appName, []string{"--tree"})
+}
+
+// InspectJSON gets the JSON output from kapp inspect
+func (c *Client) InspectJSON(appName string) (*KappInspectOutput, error) {
```

Comment:
Only inspectJSON should be used. The other Inspect methods should be removed. This will allow drafting our own table to display the resources.